cdap-ui/app/cdap/components/PluginJSONCreator/Create/PluginJSONMenu/LiveView/LiveConfigurationGroup/index.tsx

```diff
@@ -62,7 +74,7 @@ const LiveConfigurationGroupView: React.FC<ILiveConfigurationGroup> = ({ classes
    * Get pluginProperties, which is required for ConfigurationGroups component.
    */
   function getNewPluginProperties(config) {
-    if (!config['configuration-groups']) {
+    if (!config || !config['configuration-groups']) {
```

Comment:
can we use config.hasOwnProperty here instead?

Reply:
done